### DISCUSSION TITLE
use bito  jetbrain plugins 1.1.0 then ctrl+c ctrl+v not work in dialog box

### DISCUSSION THREAD

**Discussion by bebetterLWJ** [gitbito/bitoai#95] [Q&A]
for me, it's bad to update bito to 1.1.0, because I found that I cannot paste text into dialog box and cannot copy answer bito gennerated.I try to roll back version but failed, what can I do? my ide is goland 2022.3.4.

**Comment by bitoandy** [comment]
Hi @bebetterLWJ ,

We are looking into the copy, paste issue.
In the interim to go back to the previous version, you can do following:
1. Uninstall current version of Bito.
2. Download the previous version from URL: https://plugins.jetbrains.com/plugin/18289-chatgpt-gpt-4/versions
3. After downloading install by going to plugins and using "Install Plugin from Disk" option and selecting the downloaded version.
![Screenshot 2023-06-24 at 9 28 54 AM](https://github.com/gitbito/bitoai/assets/125080245/9cb71e38-9884-4fae-af2d-9af8696640e7)
